massa-models/src/operation.rs

```diff
@@ -1273,6 +1273,23 @@ impl Deserializer<Vec<SecureShareOperation>> for OperationsDeserializer {
     }
 }
 
+/// Execution status of an operation
+/// Wrap an optional bool.
+#[derive(Debug, Default)]
+pub struct OperationExecutionStatus {
```

Comment:
This structure doesn't seems to be used anywhere.

Reply:
removed